themes/default/layouts/challenge/list.html

```diff
@@ -0,0 +1,76 @@
+{{ define "hero" }}{{ partial "hero.html" (dict "title" "Challenges") }}{{ end }}
+{{ define "main" }}
+    <section class="max-w-4xl container mx-auto py-8 px-4 text-center text-lg">
+        <p>
+            We think Pulumi is an amazing IaC tool that increases developer velocity and handles the scale of the cloud with ease. Try it out for yourself by taking a Pulumi
```

Comment:
Thousands of developers think Pulumi is an amazing IaC tool that increases developer velocity and tames the complexity of modern cloud scale. Try it out for yourself by taking a Pulumi Challenge. You'll build something[...]